vorbis.go

```diff
@@ -205,8 +206,25 @@ func (m *metadataVorbis) Genre() string {
 }
 
 func (m *metadataVorbis) Year() int {
-	// FIXME: try to parse the date in m.c["date"] to extract this
-	return 0
+	// dateFormat is a string (layout) format using the standard layout motifs
+	// present in the time/format.go file https://golang.org/src/time/format.go#L88
+	var dateFormat string
+
+	switch len(m.c["date"]) {
+	case 4:
```

Comment:
Ah ok...
I choosed the most widely used pattern to decide the date format (https://en.wikipedia.org/wiki/Date_format_by_country). So most of the planet has for standard DD-MM-YEAR.

Also I didn't find anything that specify a standard format for the date in the vorbis comment documentation (or i might need to take another look).

Reply:
Also a length of 4 should be the Year. I mean why keep Day and Month in your date but not the Year ? Seems a really specific situation to me.

Reply:
Ok, this might need some changes.

After searching a bit more, the international standard format is [ISO 8601](https://en.wikipedia.org/wiki/ISO_8601). So we have to choose between most used and standard. And I guess it's obvious. I'll change this and add some more logic to find `-` chars in the date.

Reply:
Ok, I've just looked up the standard. We should implement this, and ignore everything else: https://wiki.xiph.org/VorbisComment#Date_and_time